src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,18 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
-    /**
-     * Imagine that some user wants to login to your site.
-     * You should check if user credentials (login and password) are valid or not.
-     * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
-     * @param password - user's password
-     * @return true if user by email exists and passed password is equal to user's password.
-     * Return false in any other cases.
-     */
+
+    private UserService userService = new UserService();
+
     public boolean login(String email, String password) {
-        return false;
+        User foundUser = userService.findByEmail(email);
+
+        if (foundUser == null) {
+            return false;
+        }
+        return foundUser.getEmail().equals(email)
+                && foundUser.getPassword().equals(password);
```

Comment:
do you really need to check email? If your user isn't null, that mean you have found it already by email. 
```suggestion
        return foundUser.getPassword().equals(password);
```